iac-extensions/terraform/src/test/resources/checks/ShortLogRetentionCheck/azure_app_service.tf

```diff
@@ -0,0 +1,71 @@
+resource "azurerm_app_service" "s6413-as-nc1" {
+  logs {
+    http_logs {
+      azure_blob_storage {
+        retention_in_days = 13  # Noncompliant {{Make sure that defining a short log retention duration is safe here.}}
+      }
+      file_system {
+        retention_in_days = 13  # Noncompliant {{Make sure that defining a short log retention duration is safe here.}}
+      }
+    }
+    application_logs {
+      azure_blob_storage {
+        retention_in_days = 13  # Noncompliant {{Make sure that defining a short log retention duration is safe here.}}
+      }
+    }
+  }
+}
+
+resource "azurerm_app_service" "s6413-as-c1" {
+  logs {
+    http_logs {
+      azure_blob_storage {
+        retention_in_days = 14
+      }
+      file_system {
+        retention_in_days = 14
+      }
+    }
+    application_logs {
+      azure_blob_storage {
+        retention_in_days = 14
+      }
+    }
+  }
+}
+
+resource "non_azurerm_app_service" "s6413-as-c2" {
+  logs {
+    http_logs {
+      azure_blob_storage {
+        retention_in_days = 0
+      }
+      file_system {
+        retention_in_days = 0
+      }
+    }
+    application_logs {
+      azure_blob_storage {
+        retention_in_days = 0
+      }
+    }
+  }
+}
```

Comment:
I don't think this test is useful. I believe we should replace it with 2 tests:
* 1 test to check the behavior of 0 with `azurerm_app_service`
* 1 test to check the behavior of a value such as 13 on something which is not `azurerm_app_service`